README.md

```diff
@@ -7,80 +7,62 @@
 [![Build Status](https://img.shields.io/github/actions/workflow/status/project-oak/oak/ci.yaml?branch=main&style=for-the-badge)](https://github.com/project-oak/oak/actions/workflows/ci.yaml?query=branch%3Amain)
 [![Docs](https://img.shields.io/badge/docs-rust-brightgreen?style=for-the-badge)](https://project-oak.github.io/oak)
 
-The goal of Project Oak is to create a specification and a reference
-implementation for the secure transfer, storage and processing of data.
+The goal of Project Oak is to provide infrastructure to transfer, store and
+process data in a secure and transparent way.
```

Comment:
Maybe explicitly mention "sensitive user data" to highlight the motivation

Reply:
Good point, done.